commit 3d9746bccbb50809e4c3de90ab5145a17af39aeb
Author: Ilya Tumaykin <itumaykin@gmail.com>
Date:   Thu May 25 14:38:02 2017 +0300

build: respect user choice for posix-acl configure option

Otherwise acl support is enabled automagically, which is bad.
See https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Automagic_dependencies
and https://sourceforge.net/p/aide/bugs/97/

diff --git a/configure.ac b/configure.ac
index 3598ebe..c45bbee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -450,7 +450,7 @@ AC_MSG_CHECKING(for posix-acl-support)
 AC_ARG_WITH([posix-acl],
 	[AC_HELP_STRING([--with-posix-acl],
 		[use POSIX ACLs (no checking)])],
-	[],
+	[with_posix_acl_support="$withval"],
 	[with_posix_acl_support=no]
 )

